src/enums/componentsReviewCertify.ts

```diff
@@ -6,12 +6,12 @@
  * - this list must match the order in `stateModel`
  */
 export enum ComponentsReviewCertify {
+  'special-resolution-confirm',
```

Comment:
The order matters in this enum that's why you changed it's position here right Travis?

Reply:
Yup

Reply:
What do you mean?